### ISSUE TITLE
does it support multi_labels classfication?

### ISSUE THREAD

**Issue by Bigmai-1234** [tensorflow/decision-forests#19]

**Comment by achoum** [comment]
No. Multi-label classification is not supported (i.e. training a model that predicts several independent labels).

The recommended solution is to train an individual TF-DF model for each label (possibly in parallel).

Remark that Decision Forests do not have the capability to extract hidden representations like Neural Networks do. In other words, "multi-head training" is equivalent to independent training.

Ps: Multi-class classification is supported (i.e. training a classification model on a label with more than two possible classes).

**Comment by Bigmai-1234** [comment]
got it, thanks!@ach

> No. Multi-label classification is not supported (i.e. training a model that predicts several independent labels).
> 
> The recommended solution is to train an individual TF-DF model for each label (possibly in parallel).
> 
> Remark that Decision Forests do not have the capability to extract hidden representations like Neural Networks do. In other words, "multi-head training" is equivalent to independent training.
> 
> Ps: Multi-class classification is supported (i.e. training a classification model on a label with more than two possible classes).

got it! thanks.
